Documentation/Proposals/NNNN-constrain-the-granularity-of-test-time-limit-durations.md

```diff
@@ -0,0 +1,163 @@
+# Constrain the granularity of test time limit durations
+
+* Proposal: [SWT-NNNN](NNNN-constrain-the-granularity-of-test-time-limit-durations.md)
+* Authors: [Dennis Weissmann](https://github.com/dennisweissmann)
+* Status: **Awaiting review**
+* Implementation: [apple/swift-testing#NNNNN](https://github.com/apple/swift-testing/pull/NNNNN)
+* Review: ([pitch](https://forums.swift.org/...))
+
+## Introduction
+
+Sometimes tests might get into a state (either due the test code itself or due to the code they're testing) where they don't make forward progress and hang.
```

Comment:
Stylistic nitpick: as this file becomes part of the package's documentation, please be sure to wrap at 80 columns as we do for other Markdown files.